oshmem/proc/proc.c

```diff
@@ -42,7 +42,7 @@ int oshmem_proc_init(void)
 {
     OBJ_CONSTRUCT(&oshmem_proc_lock, opal_mutex_t);
 
-    assert(sizeof(ompi_proc_t) >= sizeof(oshmem_proc_t));
+    assert(sizeof(ompi_proc_t) >= sizeof(ompi_proc_t));
```

Comment:
thanks @bosilca
I ran sed, and tested it without thoroughly checking the new code ...
will review it all tomorrow